django/contrib/auth/middleware.py

```diff
@@ -100,12 +98,14 @@ def _remove_invalid_user(self, request):
         but only if the user is authenticated via the RemoteUserBackend.
         """
         try:
-            stored_backend = load_backend(request.session.get(auth.BACKEND_SESSION_KEY, ''))
+            stored_backend = auth.load_backend(
+                request.session.get(auth.BACKEND_SESSION_KEY, '')
+            )
         except ImportError:
             # backend failed to load
             auth.logout(request)
         else:
-            if isinstance(stored_backend, RemoteUserBackend):
+            if isinstance(stored_backend, auth.backends.RemoteUserBackend):
```

Comment:
I'd revert this as I don't think it is better and it isn't making this more consistent with code elsewhere.
```suggestion
            if isinstance(stored_backend, RemoteUserBackend):
```